src/sdks/core/src/cpp/sdk/cpptest/unit/accessibilityTest.cpp

```diff
@@ -0,0 +1,153 @@
+#include "unit.h"
+
+
+class AccessibilityTest : public ::testing::Test {
+	protected:
+		JsonEngine* jsonEngine;
+        Firebolt::Error error = Firebolt::Error::None;
+
+	void SetUp() override
+	{
+		jsonEngine = new JsonEngine();
+	}
+
+	void TearDown() override
+	{
+		delete jsonEngine;
+	}
+
+    std::string fontFamilyToString(Firebolt::Accessibility::FontFamily fontFamily) {
+        std::string str = "";
+        switch(fontFamily) {
+            case Firebolt::Accessibility::FontFamily::MONOSPACED_SERIF: str = "monospaced_serif"; break;
+            case Firebolt::Accessibility::FontFamily::PROPORTIONAL_SERIF: str = "proportional_serif"; break;
+            case Firebolt::Accessibility::FontFamily::MONOSPACED_SANSERIF: str = "monospaced_sanserif"; break;
+            case Firebolt::Accessibility::FontFamily::PROPORTIONAL_SANSERIF: str = "proportional_sanserif"; break;
+            case Firebolt::Accessibility::FontFamily::SMALLCAPS: str = "smallcaps"; break;
+            case Firebolt::Accessibility::FontFamily::CURSIVE: str = "cursive"; break;
+            case Firebolt::Accessibility::FontFamily::CASUAL: str = "casual"; break;
+            default: str = "unknown";
+        }
+        return str;
+    }
+
+    std::string fontEdgeToString(Firebolt::Accessibility::FontEdge fontEdge) 
+    {
+        std::string str = "";
+        switch(fontEdge) 
+        {
+            case Firebolt::Accessibility::FontEdge::NONE: str = "none"; break;
+            case Firebolt::Accessibility::FontEdge::RAISED: str = "raised"; break;
+            case Firebolt::Accessibility::FontEdge::DEPRESSED: str = "depressed"; break;
+            case Firebolt::Accessibility::FontEdge::UNIFORM: str = "uniform"; break;
+            case Firebolt::Accessibility::FontEdge::DROP_SHADOW_LEFT: str = "drop_shadow_left"; break;
+            case Firebolt::Accessibility::FontEdge::DROP_SHADOW_RIGHT: str = "drop_shadow_right"; break;
+            default: str = "unknown";
+        }
+        return str;
+    }
+
+};
+
+TEST_F(AccessibilityTest, ClosedCaptions)
```

Comment:
What's the difference between this test and the one starting at line 85?

Reply:
This test does unit testing for `Accessibility.closedCaptions()`  method available and the other one is for `Accessibility.closedCaptionSettings()` method.